plugins/@grouparoo/hubspot/src/lib/export/exportProfile.ts

```diff
@@ -78,3 +95,14 @@ export const exportProfile: ExportProfilePluginMethod = async ({
     }
   }
 };
+
+function formatVar(value) {
+  if (value === undefined || value === null) {
+    return "";
```

Comment:
sure this shouldn't be null?

Reply:
the results are the same with "" null or undefined. All returns "" as result.

Reply:
let's switch it to `null`. generally more correct for things that are integers, booleans, etc.